container/option_test.go

```diff
@@ -0,0 +1 @@
+package container_test
```

Comment:
can we remove this folder and file?

Reply:
ping @alessio. could we get this in a follow up pr